test/fakes/fakeservernodeattestor/nodeattestor.go

```diff
@@ -57,41 +54,40 @@ func New(t *testing.T, name string, config Config) nodeattestor.NodeAttestor {
 		config: config,
 	}
 
-	v0 := new(nodeattestor.V0)
-	plugintest.Load(t, catalog.MakeBuiltIn(name, nodeattestorv0.NodeAttestorPluginServer(plugin)), v0)
+	v0 := new(nodeattestor.V1)
+	plugintest.Load(t, catalog.MakeBuiltIn(name, nodeattestorv1.NodeAttestorPluginServer(plugin)), v0)
 	return v0
 }
 
 type nodeAttestor struct {
-	nodeattestorv0.UnsafeNodeAttestorServer
+	nodeattestorv1.UnsafeNodeAttestorServer
 
 	name   string
 	config Config
 }
 
-func (p *nodeAttestor) Attest(stream nodeattestorv0.NodeAttestor_AttestServer) (err error) {
+func (p *nodeAttestor) Attest(stream nodeattestorv1.NodeAttestor_AttestServer) (err error) {
 	req, err := stream.Recv()
 	if err != nil {
 		return errs.Wrap(err)
 	}
 
-	if req.AttestationData == nil {
-		return errs.New("request is missing attestation data")
+	payload := req.GetPayload()
+	if payload == nil {
+		return errs.New("request is missing payload")
 	}
 
-	if req.AttestationData.Type != p.name {
-		return errs.New("request has wrong attestation data type: expected %q got %q", p.name, req.AttestationData.Type)
-	}
-
-	id, ok := p.config.Data[string(req.AttestationData.Data)]
+	id, ok := p.config.Payloads[string(payload)]
 	if !ok {
-		return errs.New("no ID configured for attestation data %q", string(req.AttestationData.Data))
+		return errs.New("no ID configured for attestation data %q", string(payload))
```

Comment:
nit: may we stop using errs? since we are removing it from all places and replace it for proto status?

Reply:
sgtm